### PR TITLE
remove probe.  version 2.5.3

### PR DESCRIPTION
**Pull request by djw1149**

### CODE REVIEW

**Comment by vixie** [comment]
if we remove probe, then we should make the system "dnsdb" a synonym for "dnsdb2", and add "dnsdb1" to indicate the older pre-SAF version. otherwise dnsdb2 will likely not get much exercise by users with older dnsdbq config files.

**Comment by djw1149** [comment]
Pushed switch a change.
Also restructured the man page a bit.

File: dnsdbq.man [hunk 1]
```diff
@@ -32,7 +32,7 @@
 .Op Fl l Ar query_limit
 .Op Fl M Ar max_count
 .Op Fl N Ar raw_name
-.Op Fl n Ar name
+.Op Fl n Ar name`
```
Comment (vixie) [inline dnsdbq.man:35]: why is there a backtick (`) here?

**Review by vixie — CHANGES_REQUESTED** [review]

File: dnsdbq.man (hunk 1 above)
Comment (djw1149) [reply dnsdbq.man]: A stray character.  Removed and pushed.

**Pull request merged**